private/buf/cmd/buf/buf.go

```diff
@@ -138,7 +142,7 @@ func NewRootCommand(name string) *appcmd.Command {
 				SubCommands: []*appcmd.Command{
 					convert.NewCommand("convert", builder),
 					migratev1beta1.NewCommand("migrate-v1beta1", builder),
-					studioagent.NewCommand("studio-agent", builder),
+					studioagent.NewCommand("studio-agent", noTimeoutBuilder),
```

Comment:
I found no way to modify that within the studio-agent scope, builder options/params are not accessible, and the expected input is an interface.